utilities/src/main/java/io/syndesis/qe/TestConfiguration.java

```diff
@@ -51,6 +51,10 @@
     public static final String SYNDESIS_CRD_URL = "syndesis.config.crd.url";
     public static final String SYNDESIS_OPERATOR_IMAGE = "syndesis.config.operator.image";
     public static final String SYNDESIS_CR_URL = "syndesis.config.cr.url";
+    public static final String SYNDESIS_DEFAULT_CR_VERSION = "syndesis.config.cr.version";
```

Comment:
probably I would just use `master` in the `defaultValues`, I doubt anyone will change the version by himself

Reply:
I agree.